### ISSUE TITLE
Issue downloading tracks with no Track length

### ISSUE THREAD

**Issue by zhordon** [Dniel97/orpheusdl-beatport#6]
I came across an issue with a track with no track length listed. The module fails to download it, but other sources allow it to work as intended

https://www.beatport.com/track/the-right-way/19234567

![Screenshot 2024-10-03 at 1 10 05 PM](https://github.com/user-attachments/assets/f214f621-6de1-4e75-94fb-8be4d6d8b63f)

When downloading with the module it gives the following error message

```
Traceback (most recent call last):
  
File "/Users/user/Music/OrpheusDL/orpheus.py", line 225, in <module>
    main()

File "/Users/user/Music/OrpheusDL/orpheus.py", line 220, in main
    orpheus_core_download(orpheus, media_to_download, tpm, sdm, path)

File "/Users/user/Music/OrpheusDL/orpheus/core.py", line 399, in orpheus_core_download
    downloader.download_track(media_id, extra_kwargs=media.extra_kwargs)

File "/Users/user/Music/OrpheusDL/orpheus/music_downloader.py", line 294, in download_track
    track_info: TrackInfo = self.service.get_track_info(track_id, quality_tier, codec_options, **extra_kwargs)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/Users/user/Music/OrpheusDL/modules/beatport/interface.py", line 351, in get_track_info
    duration=track_data.get('length_ms') // 1000,
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^~~~~~

TypeError: unsupported operand type(s) for //: 'NoneType' and 'int'
```

Is there a way to fix this issue?

**Comment by Dniel97** [comment]
Hello this has already been fixed in the develop branch: db07e8bcf486c18310286225924c8dee4a0989ad.

**Comment by zhordon** [comment]
I just got back from holidays. So following up on this.

It appears fixed when downloading it in track by track, but not able to download as a release.

```
User@User-Mac-Studio OrpheusDLB % python3 orpheus.py https://www.beatport.com/release/the-right-way-mark-maxwell-extended-remix/4639133

   ____             _                    _____  _      
  / __ \           | |                  |  __ \| |     
 | |  | |_ __ _ __ | |__   ___ _   _ ___| |  | | |     
 | |  | | '__| '_ \| '_ \ / _ \ | | / __| |  | | |     
 | |__| | |  | |_) | | | |  __/ |_| \__ \ |__| | |____ 
  \____/|_|  | .__/|_| |_|\___|\__,_|___/_____/|______|
             | |                                       
             |_|                                       
             
            
Traceback (most recent call last):
  File "/Users/Music/OrpheusDLB/orpheus.py", line 225, in <module>
    main()
  File "/Users/Music/OrpheusDLB/orpheus.py", line 220, in main
    orpheus_core_download(orpheus, media_to_download, tpm, sdm, path)
  File "/Users/Music/OrpheusDLB/orpheus/core.py", line 397, in orpheus_core_download
    downloader.download_album(media_id, extra_kwargs=media.extra_kwargs)
  File "/Users/Music/OrpheusDLB/orpheus/music_downloader.py", line 207, in download_album
    album_info: AlbumInfo = self.service.get_album_info(album_id, **extra_kwargs)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/Music/OrpheusDLB/modules/beatport/interface.py", line 285, in get_album_info
    duration=sum([t.get('length_ms') // 1000 for t in tracks]),
                  ~~~~~~~~~~~~~~~~~~~^^~~~~~
TypeError: unsupported operand type(s) for //: 'NoneType' and 'int'
```
https://www.beatport.com/release/the-right-way-mark-maxwell-extended-remix/4639133

Is there an alternate solution to this other than downloading as track?
